server/install.sh

```diff
@@ -64,6 +94,7 @@ if [ -f /var/www/bis/config/db_settings.ini ] ;then
 else
 	err "Database settings are not configured"
 	doing "Configuring ..."
+	mkdir /var/www/bis/config
```

Comment:
This could miss a /server/ in the path. But that's potentially fixed by #24. Just make sure the conflicts are resolved correctly.